### PR TITLE
SKU additions to products

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- Adds all the SKUs for the product in the hardware page (except unlisted)

@kateryna-lukina please double check the SKUs, especially the ones with multiples

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/01.mkr/01.boards/mkr-1000-wifi/product.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ forumCategorySlug: '/hardware/mkr-boards/mkr1000/137'
 certifications: [CE, UKCA]
 status: end-of-life
 productCode: '017'
+sku: [ABX00004]
```
Comment (kateryna-lukina) [inline content/hardware/01.mkr/01.boards/mkr-1000-wifi/product.md:12]:
```suggestion
sku: [ABX00004, ABX00011]

```

File: content/hardware/01.mkr/01.boards/mkr-zero/product.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ core: arduino:samd
 forumCategorySlug: '/hardware/mkr-boards/mkrzero/140'
 productCode: '016'
 certifications: [CE, UKCA]
+sku: [ABX00007]
```
Comment (kateryna-lukina) [inline content/hardware/01.mkr/01.boards/mkr-zero/product.md:11]:
```suggestion
sku: [ABX00007, ABX00012]

```

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md [hunk 3]
```diff
@@ -2,12 +2,13 @@
 title: Portenta Machine Control
 url_shop: https://store.arduino.cc/portenta-machine-control
 hasSetupGuide: true
-primary_button_url: /tutorials/portenta-machine-control/user-manual
-primary_button_title:  Arduino User Manual
+primary_button_url: /software/plc-ide/tutorials/plc-ide-setup-license
+primary_button_title:  PLC IDE License
```
Comment (jhansson-ard) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md:6]: @Matteo-it this is not matching the main branch :( it should be the user manual

File: content/hardware/06.nicla/boards/nicla-vision/product.md [hunk 4]
```diff
@@ -7,8 +7,9 @@ primary_button_title: User Manual
 secondary_button_url: /hardware/nicla-vision/#tutorials
 secondary_button_title: All Tutorials
 core: arduino:mbed_nicla
-certifications: [CE, UKCA, RoHS, RCM, FCC]
+certifications: [CE, UKCA, RoHS, RCM]
```
Comment (jhansson-ard) [inline content/hardware/06.nicla/boards/nicla-vision/product.md:10]: @Matteo-it this is not matching the main branch - it should contain the FCC certification

Comment (jhansson-ard) [reply content/hardware/06.nicla/boards/nicla-vision/product.md:10]: We just need to make sure your branch is updated to main before merging ⭐

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md (hunk 3 above)
Comment (jhansson-ard) [reply content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md:6]: We just need to make sure your branch is updated to main before merging ⭐

File: content/hardware/02.hero/boards/leonardo/product.md [hunk 5]
```diff
@@ -8,6 +8,7 @@ core: arduino:avr
 productCode: '008'
 forumCategorySlug: '/hardware/12'
 certifications: [CE, UKCA]
+sku: [A000057]
```
Comment (kateryna-lukina) [inline content/hardware/02.hero/boards/leonardo/product.md:11]:
```suggestion
sku: [A000052, A000057]

```

File: content/hardware/02.hero/boards/micro/product.md [hunk 6]
```diff
@@ -8,6 +8,7 @@ core: arduino:avr
 productCode: '006'
 forumCategorySlug: '/hardware/12'
 certifications: [CE, UKCA]
+sku: [A000053]
```
Comment (kateryna-lukina) [inline content/hardware/02.hero/boards/micro/product.md:11]:
```suggestion
sku: [A000053, A000093]

```

File: content/hardware/03.nano/carriers/nano-screw-terminal-adapter/product.md [hunk 7]
```diff
@@ -6,6 +6,7 @@ primary_button_url: /tutorials/nano-screw-terminal-adapter/getting-started-nano-
 primary_button_title: Get Started
 certifications: [CE, FCC, IC, RCM, REACH, RoHS, UKCA, WEEE]
 forumCategorySlug: '/hardware/12'
+sku: [ASX00037]
```
Comment (kateryna-lukina) [inline content/hardware/03.nano/carriers/nano-screw-terminal-adapter/product.md:9]:
```suggestion
sku: [ASX00037, ASX00037-3P]

```

File: content/hardware/04.pro/carriers/portenta-breakout/product.md [hunk 8]
```diff
@@ -5,6 +5,7 @@ url_guide: /tutorials/portenta-breakout/getting-started/
 primary_button_url: /tutorials/portenta-breakout/getting-started/
 primary_button_title: Get Started
 certifications: [CE, UKCA]
+sku: [AKX00031]
```
Comment (kateryna-lukina) [inline content/hardware/04.pro/carriers/portenta-breakout/product.md:8]:
```suggestion
sku: [ASX00031]
```

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md (hunk 3 above)
Comment (Matteo-it) [reply content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md:6]: I've not rebased my branch on main, i'll do it just before merging, so for now it's not updated!

File: content/hardware/10.mega/boards/due/product.md [hunk 9]
```diff
@@ -8,6 +8,7 @@ core: arduino:sam
 forumCategorySlug: '/hardware/arduino-due/64'
 productCode: '009'
 certifications: [CE, UKCA]
+sku: [A000062]
```
Comment (kateryna-lukina) [inline content/hardware/10.mega/boards/due/product.md:11]:
```suggestion
sku: [A000056, A000062]

```

**Review by kateryna-lukina — COMMENTED** [review]
Fixed a couple of missing SKUs 👍

File: content/hardware/01.mkr/01.boards/mkr-1000-wifi/product.md [hunk 10]
```diff
@@ -9,7 +9,8 @@ forumCategorySlug: '/hardware/mkr-boards/mkr1000/137'
 certifications: [CE, UKCA]
 status: end-of-life
 productCode: '017'
-sku: ABX00004
+sku: [ABX00004, ABX00011]
+
 ---
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/01.boards/mkr-1000-wifi/product.md:14]:
```suggestion
sku: [ABX00004, ABX00011]
---
```

File: content/hardware/01.mkr/01.boards/mkr-zero/product.md [hunk 11]
```diff
@@ -8,6 +8,8 @@ core: arduino:samd
 forumCategorySlug: '/hardware/mkr-boards/mkrzero/140'
 productCode: '016'
 certifications: [CE, UKCA]
+sku: [ABX00007, ABX00012]
+
 ---
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/01.boards/mkr-zero/product.md:13]:
```suggestion
sku: [ABX00007, ABX00012]
---
```

File: content/hardware/02.hero/boards/leonardo/product.md [hunk 12]
```diff
@@ -8,6 +8,8 @@ core: arduino:avr
 productCode: '008'
 forumCategorySlug: '/hardware/12'
 certifications: [CE, UKCA]
+sku: [A000052, A000057]
+
 ---
```
Comment (jhansson-ard) [inline content/hardware/02.hero/boards/leonardo/product.md:13]:
```suggestion
sku: [A000052, A000057]
---
```

File: content/hardware/02.hero/boards/micro/product.md [hunk 13]
```diff
@@ -8,6 +8,8 @@ core: arduino:avr
 productCode: '006'
 forumCategorySlug: '/hardware/12'
 certifications: [CE, UKCA]
+sku: [A000053, A000093]
+
 ---
```
Comment (jhansson-ard) [inline content/hardware/02.hero/boards/micro/product.md:13]:
```suggestion
sku: [A000053, A000093]
---
```

File: content/hardware/03.nano/carriers/nano-screw-terminal-adapter/product.md [hunk 14]
```diff
@@ -6,6 +6,8 @@ primary_button_url: /tutorials/nano-screw-terminal-adapter/getting-started-nano-
 primary_button_title: Get Started
 certifications: [CE, FCC, IC, RCM, REACH, RoHS, UKCA, WEEE]
 forumCategorySlug: '/hardware/12'
+sku: [ASX00037, ASX00037-3P]
+
 ---
```
Comment (jhansson-ard) [inline content/hardware/03.nano/carriers/nano-screw-terminal-adapter/product.md:11]:
```suggestion
sku: [ASX00037, ASX00037-3P]
---
```

**Review by kateryna-lukina — APPROVED** [review]
LGTM ⭐

**Comment by jhansson-ard** [comment]
@Matteo-it will you help with the conflicts? 🙏🏼

**Comment by Matteo-it** [comment]
> @Matteo-it will you help with the conflicts? 🙏🏼

@jhansson-ard done! Let me know when i can merge it

**Review by Matteo-it — APPROVED** [review]

**Comment by jhansson-ard** [comment]
You can merge away!

**Pull request merged**